src/ZPublisher/HTTPRequest.py

```diff
@@ -1493,6 +1501,53 @@ def __init__(self, fp, environ):
             add_field(field)
 
 
+class LimitedFileReader:
+    """File wrapper emulating EOF."""
+
+    # attributes to be delegated to the file
+    DELEGATE = set("close closed fileno mode name".split())
```

Comment:
```suggestion
    DELEGATE = set(["close", "closed", "fileno", "mode", "name"])
```

Using `split` it is easier to write but harder to read, because it is necessary to read the whole line to understand that not a string is given to `set` but a list.